.github/workflows/test.yml

```diff
@@ -8,6 +8,10 @@ on:
       - main
   workflow_dispatch:
 
+env:
+  LSL_RELEASE_URL: "https://github.com/sccn/liblsl/releases/download/v1.16.2"
+  LSL_RELEASE: "1.16.2"
+
```

Comment:
This should be removed.

Reply:
Oops! Thanks. I did a force push to amend the previous 'remove playback test' commit.